packages/netlify/test/functions/redirects.test.js

```diff
@@ -13,8 +14,10 @@ describe('SSR - Redirects', () => {
 	it('Creates a redirects file', async () => {
 		const redirects = await fixture.readFile('./_redirects');
 		const parts = redirects.split(/\s+/);
-		expect(parts).to.deep.equal(['', '/other', '/', '301', '']);
-		expect(redirects).to.matchSnapshot();
+		assert.deepEqual(parts,['', '/other', '/', '301', '']);
+		// TODO: Snapshots are not supported in Node.js test yet (https://github.com/nodejs/node/issues/48260)
+		// expect(redirects).to.matchSnapshot();
```

Comment:
could we inline the snapshot, and do `assert.equal(redirects, "....")`?

Reply:
I'll give it a bash. 🙇 thank you for the feedback!

In that case we won't continue generating snapshots, which should be okay if this pages content doesn't change often?